.github/workflows/ci.yaml

```diff
@@ -37,7 +43,186 @@ jobs:
         with:
           toolchain: stable
           override: true
-      - uses: actions-rs/cargo@v1
+      - name: cargo fetch
+        uses: actions-rs/cargo@v1
+        with:
+          command: fetch
+      - name: cargo test build
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --tests --release
+      - name: cargo test
+        uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --all-features
```

Comment:
Same here, testing all features is important

Reply:
Ditto.